app/gui/src/ide.rs

```diff
@@ -51,7 +51,7 @@ impl Ide {
     /// Constructor.
     pub async fn new(
         application: Application,
-        view: ide_view::project::View,
+        view: ide_view::root::View,
```

Comment:
I don't think that 'root' is a good name. The name 'project' explains what to expect there. 'Root' can be just anything. It's too broad.

Reply:
Will `main` be a better name? We can also reexport it as `ide_view::View`, but it won't eliminate the need in some module name.

Reply:
Let's seep it then!